src/components/interaction-designer/Connection.vue

```diff
@@ -181,16 +182,17 @@ export default {
     mouseOverHandler() {
       this.line.setOptions(this.prominentOptions)
       this.activateConnection({connectionContext: this.connectionContext})
+      this.$emit('isLineHovered', true)
     },
     mouseOutHandler() {
       if (!this.isPermanentlyActive) {
         this.line.setOptions(this.options)
         this.deactivateConnection({connectionContext: this.connectionContext})
       }
+      this.$emit('isLineHovered', false)
```

Comment:
Do you think we should rename these? I found it a bit confusing at first glance in the consuming Block vue file because they're named as boolean flags rather than verbs/actions. 

An idea is to have two separate events matching convention of in/out: `lineMouseIn` + `lineMouseOut`— This also mitigates the need to provide the boolean. Thoughts?

Reply:
I think renaming makes sense and made these changes.